docs/mining.md

```diff
@@ -23,10 +21,18 @@ first_attempt_time_ms = 1000
 # Time to spend on subsequent attempts to make a block, in milliseconds.
 # This can be bigger -- new block-commits will be RBF'ed.
 subsequent_attempt_time_ms = 60000
```

Comment:
I believe this field and `first_attempt_time_ms` no longer apply in epoch 3.0 and can probably be removed from here.

Reply:
`first_attempt_time_ms` is still valid the last time i checked, but i'll remove it beb3e63f19e77b5db435e98fa1a6d7aac3b84cca

Reply:
[Some context here](https://nakamotoslack.slack.com/archives/C06N3DE9ARM/p1729527107210219?thread_ts=1729526549.215069&cid=C06N3DE9ARM)

Reply:
ack. removed it